README.md

```diff
@@ -31,14 +35,35 @@ unsubscribe('b', h)
 ```
 
 ## Naming convention
+
 To stay consistent, we encourage you to use the following syntax when declaring events
 
 `app-id:object:verb`
 
 ### Examples:
-- nextcloud:unified-search:closed
-- files:file:uploading
-- files:file:uploaded
-- contacts:contact:deleted
-- calendar:event:created
-- forms:answer:updated
+
+- `nextcloud:unified-search:closed`
+- `files:file:uploading`
+- `files:file:uploaded`
+- `contacts:contact:deleted`
+- `calendar:event:created`
+- `forms:answer:updated`
+
+## Development
+
+```sh
+npm install
+
+npm run build
+npm run test
+```
+
+### Requirements
+
+- [Node 14 or higher](https://nodejs.org/en/download/)
+- [NPM 7 or higher](https://www.npmjs.com/package/npm)
+
+### Recommendations
+
+- [Node 16](https://nodejs.org/en/download/)
+- [NPM 8](https://www.npmjs.com/package/npm)
```

Comment:
We should add release documentation after https://github.com/nextcloud/nextcloud-event-bus/pull/490/files#r873962228 is clarified.

Reply:
I guess we can move forward.
https://github.com/nextcloud/standards/discussions/6